apps/lightning-terminal/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.7"
 
 services:
   web:
-    image: lightninglabs/lightning-terminal:v0.5.0-alpha@sha256:1c9c59fe0316de5d64e89e205f91fb647660a865428daacfe582cb968fd42f87
+    image: lightninglabs/lightning-terminal:v0.5.2-alpha@sha256:fa6415256bbac022784c2f555dbf1443da0df65cbad742dc433e00afd00841f1
```

Comment:
According to [dockerhub](https://hub.docker.com/layers/lightninglabs/lightning-terminal/v0.5.2-alpha/images/sha256-31cd51c7e65325f7360a9fa97ddce2d1695859be143048d9763ecdded6f150cb?context=explore) the sha256 for the container is 

- `31cd51c7e65325f7360a9fa97ddce2d1695859be143048d9763ecdded6f150cb` for amd64
- `fcfbe6dcac99121fa68fabef75a9898a60f2b9c3cbcd1f65d10479768254c925` for arm64